### DISCUSSION TITLE
Can I Join The Team?

### DISCUSSION THREAD

**Discussion by ImSurajx** [ROM-builders/temporary#10656] [Q&A]
Here Is Some Roms That' I Builds :)

Pixel Experience :)
https://cirrus-ci.com/build/5049382557974528

Pixel Experience Plus :)
https://cirrus-ci.com/build/5891016799551488

Arrow Os Gapps :)
https://cirrus-ci.com/build/5891016799551488

Cr Droid ~  Gapps & Vanilla
https://cirrus-ci.com/build/5822852883546112
https://cirrus-ci.com/build/6210864121380864

Project Arcana :)
https://cirrus-ci.com/build/5985172414791680

Device Tree That I Use :) Please!  Also Check Different Branches Please :)
https://gitlab.com/ImSurajKr/device_xiaomi_lavender

#learn Yes I Learned Everything That Was Written In It.
 
Old Information : From Old GitHub Account I Build More Than 5 Sucessfull Builds But I Deleted That Account Because I Want To Create New Account With New Gmail For Android Development Or  i  Created This Account Recently Mean 15 Days Ago.Here I Also Make Many Successfull Build With The Help Of Cirrus.

GitHub Profile :) 
https://github.com/ImSurajx

Telegram ID :)
https://t.me/ImSurajx

GitLab ID :)
https://gitlab.com/ImSurajKr

ThankYou Very Much To Providing Us This Awesome Features.

**Comment by ImSurajx** [comment]
@Apon77 When Give Ans Sir?

**Reply by Apon77** [reply]
you need to wait few days until we found that you learned #learn things properly. otherwise i cant help.
